media-gfx/ahoviewer/ahoviewer-1.6.2.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools gnome2-utils xdg-utils
+
+DESCRIPTION="A GTK2 image viewer, manga reader, and booru browser"
+HOMEPAGE="https://github.com/ahodesuka/ahoviewer"
+SRC_URI="https://github.com/ahodesuka/ahoviewer/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gnutls +gstreamer libsecret +rar +ssl +zip"
+
+DEPEND="
+	dev-cpp/glibmm:2
+	dev-cpp/gtkmm:2.4
+	dev-libs/libconfig[cxx]
+	dev-libs/libxml2:2
+	dev-libs/libsigc++:2
+	net-misc/curl
+	gnutls? (
+		net-libs/gnutls:=
+		net-misc/curl[curl_ssl_gnutls]
+	)
+	gstreamer? (
+		media-libs/gst-plugins-bad:1.0
+		media-libs/gstreamer:1.0
+	)
+	libsecret? ( app-crypt/libsecret )
+	rar? ( app-arch/unrar )
+	ssl? (
+		dev-libs/openssl:=
```

Comment:
Is `:0.9.8` slot going to work here?

Reply:
No :) I thought it would, earlier, but should had **checked** it. Sorry, fixing now.